back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -39,7 +43,9 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     :linear_scale_label_3_multiloc,
     :linear_scale_label_4_multiloc,
     :linear_scale_label_5_multiloc,
-    :linear_scale_label_6_multiloc,
+    if: proc { |object, _params| object.input_type == 'sentiment_linear_scale' || object.linear_scale? || object.supports_matrix_statements? }
+
+  attributes :linear_scale_label_6_multiloc,
     :linear_scale_label_7_multiloc,
     :linear_scale_label_8_multiloc,
```

Comment:
This was strange - I had to separate out the `linear_scale_label` 1 through 5 and 6 through 11 for some reason, or else the serialization didn't work properly for `linear_scale` or `sentiment_linear_scale` input types :thinking: It was like it overwrote the attributes when I tried to use two separate ones.

Any suggestions for how to clean this up?